schemas/v1/19_actor_mappings.go

```diff
@@ -0,0 +1,30 @@
+package v1
+
+func init() {
+	patches.Register(
+		19,
+		`
+CREATE TABLE IF NOT EXISTS {{ .SchemaName | default "public"}}.actor_codes (
+	cid		text  NOT NULL,
+	code	text  NOT NULL,
+	
+	PRIMARY KEY (cid, code)
+);
+COMMENT ON TABLE {{ .SchemaName | default "public"}}.actor_codes IS 'A mapping of a builtin actors CID to a human friendly name.';
+COMMENT ON COLUMN {{ .SchemaName | default "public"}}.actor_codes.cid IS 'CID of the actor from builtin actors.';
+COMMENT ON COLUMN {{ .SchemaName | default "public"}}.actor_codes.code IS 'Human-readable identifier for the actor.';
+    
+CREATE TABLE IF NOT EXISTS {{ .SchemaName | default "public"}}.actor_methods (
+	family		text  NOT NULL,
```

Comment:
I think it would be worth including the actor Code (CID) here as well. Perhaps we could include the first actor code a method is valid for?

Reply:
> I think it would be worth including the actor Code (CID) here as well. Perhaps we could include the first actor code a method is valid for?


The actor code cid differs by network but the method numbers are consistent across networks. We also often don't join by the actor code cid in analyses. If we decide to add actor code CIDs, we'd have rows with multiple values duplicated and the additional information is not more useful.